sources/vega/governance.proto

```diff
@@ -86,8 +86,10 @@ message NewMarket {
 
 // Update an existing market on Vega
 message UpdateMarket {
+  // The market to update
+  string market_id = 1;
   // The updated configuration of the market
-  UpdateMarketConfiguration changes = 1;
+  UpdateMarketConfiguration changes = 2;
 }
```

Comment:
Why not add it inside the changes directly?

Reply:
I wanted to keep the market configuration focus on the actual market change, and not the additional data that help to retrieve the market